JSON_serializer/main.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import argparse
+import sys
+import os.path
+
+
+import serializer
+
+
+def create_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-f', dest='file_path', nargs='*', type=str,
+                        help='decode file of defined directory '
+                             '(which contains data in json format) from json to object')
+    parser.add_argument('-u', dest='url', nargs='*', required=False, type=str,
+                        help='decode data defined in URL from json to object ')
+    parser.add_argument('-j', dest='json_string', nargs='*', required=False, type=str,
+                        help='decode single quote string in json format from json to object')
+    parser.add_argument('unknown_data', nargs='*', type=str, help='call instruction')
+    return parser
+
+
+def is_valid_file(arg):
+    if not os.path.exists(arg):
+        print(f'The file {arg} does not exist. Check your input data.')
+        return False
+    else:
+        return True
+
+
+def is_url(address):
+    try:
+        url = requests.head(address)
+        return url.status_code == 200
+    except IOError:
+        print(f'The url {address} is not correct. Check your input data. ')
```

Comment:
This error message is too implicit. User will see this message when he/she has problems with connection, for example. Lack of internet connection doesn't mean that URL is incorrect, isn't it?